book/src/values.md

```diff
@@ -42,7 +46,10 @@ $$[b_0,\dots,b_{N-1}]\mapsto \sum_{i=0}^{N-1} b_i \cdot 2^{8i},$$
 
 and $\jmath_\texttt{string->bytes}$ is the canonical mapping of a string to its UTF-8 representation.
 
+## `Raw`
+"Raw" is short for "raw value".  A `Raw` exposes a backend value on the frontend.
 
+With the plonky2 backend, a `Raw` is a tuple of 4 elements of the Goldilocks field.
```

Comment:
is the `Raw` in this section equivalent to the `Value` type at the section `backendtypes.md`?